src/modules/dashboard/components/CreateColonyWizard/TokenSelector.tsx

```diff
@@ -107,19 +112,21 @@ const TokenSelector = ({
         onTokenSelectError(false);
       }
     },
-    [onTokenSelect, onTokenSelectError],
+    [onTokenSelect, onTokenSelectError, onCheckingAddress],
   );
 
   const handleGetTokenError = useCallback(
     (error: Error) => {
       setLoading(false);
+      onCheckingAddress(false);
```

Comment:
you can see here and in other places that "onCheckingAddress" has the same meaning as "loading" so we actually don't need this new method.

Reply:
Agreed:  "onCheckingAddress" has the same meaning as "loading"  - I refactored, and also moved the isCheckingAddress functionality into "onTokenSelect".